docs/modules/ROOT/pages/advanced-how-tos/managing-compliance-with-ec/identifying-policy-in-use.adoc

```diff
@@ -0,0 +1,27 @@
+= Identifying Policy in-use
```

Comment:
I feel like this title doesn't represent the content on the page. This page is describing where and how policies are specified in Konflux. Can we change the title to something like "Policy evaluations in {ProductName}"?

Reply:
Done.